pkg/config/reminder/config.go

```diff
@@ -52,3 +60,14 @@ func RegisterReminderFlags(v *viper.Viper, flags *pflag.FlagSet) error {
 
 	return registerRecurrenceFlags(v, flags)
 }
+
+func validateEventConfig(cfg serverconfig.EventConfig) error {
+	switch cfg.Driver {
+	case constants.NATSDriver:
+	case constants.SQLDriver:
+	default:
+		return fmt.Errorf("events.driver %s is not supported", cfg.Driver)
+	}
+
+	return nil
+}
```

Comment:
This prevents using e.g. the flag driver.  I think that's okay, but I'd tend to write this as a deny-list of configuration that we know won't work rather than an allow-list (because we'll need to go back and expand the allow-list as we add new messaging configurations).